powerauth-data-adapter/src/main/java/io/getlime/security/powerauth/app/dataadapter/api/DataAdapter.java

```diff
@@ -136,6 +148,22 @@
      */
     CreateSmsAuthorizationResponse createAndSendAuthorizationSms(String userId, String organizationId, AccountStatus accountStatus, AuthMethod authMethod, OperationContext operationContext, String lang) throws InvalidOperationContextException, DataAdapterRemoteException;
 
+    /**
+     * Send an authorization SMS message with generated authorization code.
+     * @param userId User ID.
+     * @param organizationId Organization ID.
+     * @param accountStatus User account status.
+     * @param authMethod Authentication method.
+     * @param operationContext Operation context.
+     * @param messageId Message ID.
+     * @param authorizationCode Authorization code.
+     * @param lang Language for localization.
+     * @return Message ID.
+     * @throws InvalidOperationContextException Thrown when operation context is invalid.
+     * @throws DataAdapterRemoteException Thrown when remote communication fails or SMS message could not be delivered.
+     */
+    SendAuthorizationSmsResponse sendAuthorizationSms(String userId, String organizationId, AccountStatus accountStatus, AuthMethod authMethod, OperationContext operationContext, String messageId, String authorizationCode, String lang) throws InvalidOperationContextException, DataAdapterRemoteException;
```

Comment:
We should probably rename the `createAndSendAuthorizationSms` method since we no longer expect it to send the SMS message, right? If I did not understand this correctly, we should probably extend the JavaDoc comments a bit to explain what the `createAndSendAuthorizationSms` and `sendAuthorizationSms` methods do...